packages/edit-post/src/editor.js

```diff
@@ -164,7 +164,7 @@ function Editor( {
 	const styles = useMemo( () => {
 		const themeStyles = [];
 		const presetStyles = [];
-		settings.styles.forEach( ( style ) => {
+		forEach( settings.styles, ( style ) => {
```

Comment:
I decided to use `lodash.forEach` helper for convenience. It can handle `array`, `object`, and cases where value is falsy.